### PR TITLE
Adds a warning to the Readme about harassment and complying with the Github Terms of Service

### PR DESCRIPTION
**Pull request by Iamgoofball**
I and other contributors have become very fed up with the inappropriate and unprofessional actions of people commenting on pull requests, with the intentions of harassing the contributor because they disagree with their opinions on balance or gameplay or whether they "play the game".

(Note: I'm not talking about people posting hard stats to disprove my or other contributor's arguments, that's fine, but be respectful about it)

This is simply unacceptable behavior and I am getting very tired of having to contact our maintainers to delete 10+ comments every time I make a pull request that is a controversial balance change.

If you cannot argue your point without falling back to below the belt insults and harassment of the person opening a pull request, you will have your rights to this service revoked if you're found to be violating the Terms of Service Github requests we follow.

Github does not do timed bans, and neither does the repository, because Github is a website for **adults** to collaborate on code projects.

This message is added to the readme along with a helpful link to the Terms of Service to hopefully ensure that people will stop their harassing actions and keep their discussions civil.

### CODE REVIEW

**Comment by praisenarsie** [comment]
Wouldn't this be a better with or in a code of conduct?

**Comment by Kraseo** [comment]
(the point of telling you to play the game is to make your own unbiased opinions of game balance, not arbitrarily pulling numbers out of your ass)

**Comment by Iamgoofball** [comment]
That's what the Terms of Service is supposed to be.

@Kraseo We're not talking about that, we're talking about situations like recently where cyberboss had to delete like 8 comments aimed to harass me via making falsified claims and not contributing anything to the actual balance discussion. Please try to stay on topic.

**Comment by ChangelingRain** [comment]
If you think this PR isn't reasonable, you may be part of the problem.

File: README.md [hunk 1]
```diff
@@ -9,6 +9,9 @@
 **Wiki** https://tgstation13.org/wiki/Main_Page <BR>
 **IRC:** irc://irc.rizon.net/coderbus or if you dont have an IRC client, you can click  [here](https://kiwiirc.com/client/irc.rizon.net:6667/?&theme=cli#coderbus).<BR>
 
+## We have a zero tolerance policy on harassment of contributors and community members on our Github. This is not 4chan, this is not the game server, and this is not the forums. You will follow the Github Terms of Service. Harassment of users on our repository will result in your postings being deleted and reports being filed with Github Support to request a suspension of your account.
```
Comment (PJB3005) [inline README.md]: Github needs to be capitalized as GitHub.

**Review by PJB3005 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Iamgoofball) [reply README.md]: whoops, ty

**Comment by PKPenguin321** [comment]
Citing connection statistics is not a mindless accusation that you don't play, it's an informed statement with evidence. It is not used without reason, it is used to show that your reasoning for certain changes is faulty.

**Comment by Iamgoofball** [comment]
@PKPenguin321 We're not talking about you trying to use informed statistics to provide discussion on a balance pull request, that's not a ToS violation.

Yakumo posting issues blaming me in the title and calling me retarded is a ToS violation, however.

https://github.com/tgstation/tgstation/issues/37661

**Comment by PKPenguin321** [comment]
Then perhaps you should remove the misleading statement from the first paragraph in your OP regarding you not playing the game.

**Comment by Iamgoofball** [comment]
I'll edit it, as I meant to point out issues of "LOL he doesnt play the game fuckin ban him from github"

that doesn't actually contribute, isnt backed by facts, and is generally a dick move with the intent to harass

You posting connection info saying "hey I don't think he's qualified to provide feedback because (insert hard stats here)" is a separate thing that isn't being a dick and isn't harassment

**Comment by Cyberboss** [comment]
This sort of thing belongs in a code of conduct

**Comment by Qbopper** [comment]
counterpoint: you have done things that should have gotten you gitbanned many times that are unrelated to "does not play the game"

I am floored that /goofball/ is the one trying to make this pr - you've abused github multiple times and if I'm not mistaken posted images that would likely violate the terms of service

Don't be surprised when you get flak for trying to play the victim when you yourself have been the one dishing out worse in the past

**Comment by Kromgar** [comment]
Goofball is trying to use the pretense of cyberbullying to silence critics

**Comment by Iamgoofball** [comment]
That's complete nonsense and you know it.

You can criticize a PR respectfully and maturely.

**Pull request closed without merging**